napari/plugins/io.py

```diff
@@ -16,7 +18,7 @@ def read_data_with_plugins(
     returned, or no readers are found.
 
     Exceptions will be caught and stored as PluginErrors
-    (in plugin_manager._exceptions)
+    (in plugins.PLUGIN_ERRORS)
```

Comment:
I kinda prefer this being in the plugin manager... This is a mild preference, but could you tell us why you decided to make it a global variable instead?

Reply:
I agree with your mild preference.  I did this mostly because I had lots of issues with circular imports...  also: anything can raise a `PluginError` (not just the plugin_manager), so it seemed ok to extract that concept.  I just briefly tried to put it back on the manager, and would need to rearrange a fair amount of things.  can figure it out though if you want it

Reply:
let's figure it out in a later PR if needed. =)